### PR TITLE
Luffm commander hometown and program

### LINKED ISSUE #5: What's your favourite food?
## Favourite food?
- [x] Chocolate
- [x] Crisps
- [x] Fruit
- [x] Vegetables
- [x] Meat

### PR DESCRIPTION
**Pull request by luffm**
Closes #5

### CODE REVIEW

File: hello_world.basic [hunk 1]
```diff
@@ -0,0 +1,2 @@
+10 PRINT "HELLO WORLD!"
```
Comment (luffm) [inline hello_world.basic]: Wow. This is a great program ☆★☆

**Review by kenwayp — COMMENTED** [review]
Can you add some comments?

**Review by kenwayp — CHANGES_REQUESTED** [review]
Can you add some comments?

**Review by kenwayp — APPROVED** [review]
Looks good!

**Pull request merged**